cmd/register/register_util.go

```diff
@@ -315,29 +315,30 @@ func hydrateSpec(message proto.Message, sourceCode string, config rconfig.FilesC
 	switch v := message.(type) {
 	case *admin.LaunchPlan:
 		launchPlan := message.(*admin.LaunchPlan)
-		hydrateIdentifier(launchPlan.Spec.WorkflowId, config.Version)
```

Comment:
I think earlier we decided that we will no use config directly, We will use parameter.